core/modules/order/components/UserOrdersHistory.ts

```diff
@@ -26,6 +27,14 @@ export default {
   methods: {
     onBottomScroll () {
       ++this.pagination.current
-    }
+      const totalCount = this.$store.state.user.orders_history.totalCount ? this.$store.state.user.orders_history.totalCount : 0
+      const isLastPage = this.pagination.current > Math.ceil(totalCount / this.pagination.perPage);
+      if (!isLastPage) {
+        this.appendNewOrders({ currentPage: this.pagination.current, refresh: true })
```

Comment:
1. I think that `refresh: true` is unnecessary.  
2. Argument type `{ refresh: boolean, currentPage: any }` is not assignable to parameter type `{ resolvedFromCache: any, pageSize?: any, currentPage?: any }` 

![Zrzut ekranu 2021-04-22 o 12 07 38](https://user-images.githubusercontent.com/40273258/115696863-58182800-a363-11eb-84da-64f187d92d43.png)